src/local.rs

```diff
@@ -273,9 +273,17 @@ pub async fn run_node(
         safenode_path: Some(launcher.get_safenode_path()),
     });
 
-    Ok(Multiaddr::from_str(&format!(
-        "/ip4/127.0.0.1/tcp/{port}/p2p/{peer_id}"
-    ))?)
+    if cfg!(feature = "tcp") {
+        Ok(Multiaddr::from_str(&format!(
+            "/ip4/127.0.0.1/tcp/{port}/p2p/{peer_id}"
+        ))?)
+    } else
+    // efault to quic
```

Comment:
Typo. Maybe also use `#[cfg(feature = "quic")]` like below to keep it consistent.

Reply:
That should also force this library to be compiled with either `quic` or `tcp`, because it's not possible to compile without either feature.